docs/vc-http-api.yml

```diff
@@ -592,6 +651,91 @@ components:
           $ref: "#/components/schemas/PresentCredentialOptions"
     ProvePresentationResponse:
       $ref: "#/components/schemas/VerifiablePresentation"
+    NotifyPresentationAvailableRequest:
+      type: object
+      properties:
+        query:
+          type: object
+          description: See https://w3c-ccg.github.io/vp-request-spec/#format
+          properties:
+            type:
+              type: string
+              description: "The type of query the server should reply with."
+            credentialQuery:
+              type: object
+              description: "Details of the client's available presentation"
+
+      example:
+        {
+          "query":
+            [
+              {
+                "type": "RequestQueryByFrame",
+                "credentialQuery":
+                  [
+                    {
+                      "type":
+                        [
+                          "VerifiableCredential",
+                          "CommercialInvoiceCertificate",
+                        ],
+                      "reason": "Wallet XYZ is ready to selectively disclose new credentials.",
+                    },
+                  ],
+              },
+            ],
+        }
+
+    NotifyPresentationAvailableResponse:
+      type: object
+      properties:
+        query:
+          type: object
+          description: See https://w3c-ccg.github.io/vp-request-spec/#format
+        domain:
+          type: string
+          description: See https://w3id.org/security#domain
+        challenge:
+          type: string
+          description: See https://w3id.org/security#challenge
+      example:
```

Comment:
I was thinking there needs to be some sort of 'session' provided here, so there can be a co-relation on the Presentation endpoint; but then there is suddenly (potentially) a lot of state for this holder (server) to manage. @OR13 is there a model you have in mind how this holder (server) would link a presentation to a notification, or is that important for your use case? (It is important for some personal credential use cases).

Reply:
forgive me if this is an anti-pattern or just a dumb idea for 13,152 reasons, but might an authN token be enough of a correlation or session to manage? if it's a bad idea or deliberately out of scope, some non-normative guidance might be good to make sure it's "at your own risk" to bind challenges to authN tokens or other kinds of session tracking :D

Reply:
auth token could work, and depending on its structure (bearer token) can alleviate the need for server side session state. But I agree, we need clarity on if this is in scope for this API or not, and then language that states these assumptions.

Reply:
again these are pulled from the VC Data Model, and would seem to be non negotiable, regardless of additional transport of message level security mechanisms.

Reply:
Ok, it seems like there is alignment to put the concept of "session" or authorization etc... in a different PR?

Reply:
@mavarley I think there is alignment that authorization for all these endpoints need to be addressed consistently. I'd suggest we keep that discussion in issues until we have a clear path forward.